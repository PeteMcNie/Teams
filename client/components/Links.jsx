import React from 'react'
import { Link } from 'react-router-dom'

const Links = () => {
  return (
    <div className='links'>
      <div className='link'>
        <Link to={`/createteams`}>Create Teams</Link>
      </div>
      <div className='link'>
        <Link to={`/addplayers`}>Add New Person</Link>
      </div>
    </div>
  )
}

export default Links
