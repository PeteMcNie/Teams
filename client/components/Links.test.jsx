import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Links from './Links'

test('Links component matches snapshot', () => {
  const { container } = renderWithRedux(<Links />)
  expect(container).toMatchSnapshot()
})
