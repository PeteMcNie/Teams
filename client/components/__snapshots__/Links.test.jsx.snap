// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links component matches snapshot 1`] = `
<div>
  <div
    class="links"
  >
    <div
      class="link"
    >
      <a
        href="/createteams"
      >
        Create Teams
      </a>
    </div>
    <div
      class="link"
    >
      <a
        href="/addplayers"
      >
        Add New Person
      </a>
    </div>
  </div>
</div>
`;
